google/cloud/storage/doc/storage-main.dox

```diff
@@ -271,8 +271,8 @@ The library automatically retries requests that fail with transient errors, and
 follows the
 [recommended practice](https://cloud.google.com/storage/docs/exponential-backoff)
 to backoff between retries. Application developers can override the default
-[retry](@ref google::cloud::storage::v0::RetryPolicy) and
-[backoff](@ref google::cloud::storage::v0::BackoffPolicy) policies:
+[retry](@ref google::cloud::storage::v1::RetryPolicy) and
```

Comment:
Question: After this is merged, I'm assuming the client library API reference documentation will be updated from `v0` to `v1`. If so, please ignore and merge.

Reply:
Yes, it will.